cmake/Tools/Platform/Android/unit_test_android_deployment.py

```diff
@@ -586,14 +586,14 @@ def test_update_device_file_timestamp(tmpdir):
 
 @pytest.mark.parametrize(
     "test_config, test_package_name, test_device_storage_path", [
-        pytest.param('profile', 'com.amazon.lumberyard.foo', '/data/fool_storage'),
-        pytest.param('debug', 'com.amazon.lumberyard.foo', '/data/fool_storage'),
-        pytest.param('profile', 'com.amazon.lumberyard.bar', '/data/fool_storage'),
-        pytest.param('debug', 'com.amazon.lumberyard.bar', '/data/fool_storage'),
-        pytest.param('profile', 'com.amazon.lumberyard.foo', '/data/fool_storage2'),
-        pytest.param('debug', 'com.amazon.lumberyard.foo', '/data/fool_storage2'),
-        pytest.param('profile', 'com.amazon.lumberyard.bar', '/data/fool_storage2'),
-        pytest.param('debug', 'com.amazon.lumberyard.bar', '/data/fool_storage2')
+        pytest.param('profile', 'com.amazon.o3de.foo', '/data/fool_storage'),
+        pytest.param('debug', 'com.amazon.o3de.foo', '/data/fool_storage'),
+        pytest.param('profile', 'com.amazon.o3de.bar', '/data/fool_storage'),
+        pytest.param('debug', 'com.amazon.o3de.bar', '/data/fool_storage'),
+        pytest.param('profile', 'com.amazon.o3de.foo', '/data/fool_storage2'),
+        pytest.param('debug', 'com.amazon.o3de.foo', '/data/fool_storage2'),
+        pytest.param('profile', 'com.amazon.o3de.bar', '/data/fool_storage2'),
+        pytest.param('debug', 'com.amazon.o3de.bar', '/data/fool_storage2')
```

Comment:
These still all have 'amazon'. `com.amazon.o3de.*` can be replaced with `org.o3de` instead

Reply:
Done.